app/src/test/java/org/simple/clinic/registration/phone/loggedout/LoggedOutOfDeviceDialogLogicTest.kt

```diff
@@ -101,7 +98,9 @@ class LoggedOutOfDeviceDialogLogicTest {
     assertThat(thrownError).isNotNull()
   }
 
-  private fun setupController() {
+  private fun setupController(errorHandler: Consumer<Throwable>?) {
+    RxJavaPlugins.setErrorHandler(errorHandler)
```

Comment:
Can we also clear the error handler in the `@After` test fixture since this will affect RxJava in the entire test suite?

Reply:
Done